wikidict/utils.py

```diff
@@ -471,8 +471,7 @@ def _convert_math(expr: str, packages: List[str] = []) -> str:
     """Convert mathematics symbols to a base64 encoded GIF file."""
     from base64 import b64encode
     from io import BytesIO
-
-    from PIL import Image
```

Comment:
We can get rid of PIL from requirements then, right?

Reply:
Apparently Preview uses PIL under the wood ?

Reply:
Let's get this merged, and do eventual cleanup later :)